Makefile

```diff
@@ -24,7 +24,7 @@ dist: all
 	mkdir -p dist/atmosphere/contents/$(BLUETOOTH_MITM_TID)/flags
 	touch dist/atmosphere/contents/$(BLUETOOTH_MITM_TID)/flags/boot2.flag
 	
-  	cp toolbox.json dist/atmosphere/contents/$(BLUETOOTH_MITM_TID)/toolbox.json
+  	cp $(BLUETOOTH_MITM_TID)/toolbox.json dist/atmosphere/contents/$(BLUETOOTH_MITM_TID)/toolbox.json
```

Comment:
`$(BLUETOOTH_MITM_TID)` -> `bluetooth-mitm`

Reply:
very well, I thought that this way it would be more scalable.

Reply:
This folder doesn't exist though. The title ID is only used for the directory on the switch side. You're copying toolbox.json from the bluetooth-mitm folder you just moved it to